src/App.tsx

```diff
@@ -47,49 +62,106 @@ export function getReorderedGoods(
 //   sortType: SortType,
 // };
 
-export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button
-          type="button"
-          className="button is-info is-light"
-        >
-          Sort alphabetically
-        </button>
-
-        <button
-          type="button"
-          className="button is-success is-light"
-        >
-          Sort by length
-        </button>
-
-        <button
-          type="button"
-          className="button is-warning is-light"
-        >
-          Reverse
-        </button>
-
-        <button
-          type="button"
-          className="button is-danger is-light"
-        >
-          Reset
-        </button>
-      </div>
+type State = {
+  sortType: SortType;
+  isReversed: boolean;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    sortType: SortType.NONE,
+    isReversed: false,
+  };
+
+  handleReverse = () => {
+    this.setState((state) => ({
+      isReversed: !state.isReversed,
+    }));
+  };
+
+  handleSortByAlphabet = () => this.setState({
+    sortType: SortType.ALPHABET,
+  });
+
+  handleSortByLength = () => this.setState({
+    sortType: SortType.LENGTH,
+  });
+
+  handleReset = () => this.setState({
+    sortType: SortType.NONE, isReversed: false,
+  });
+
+  render() {
+    const { sortType, isReversed } = this.state;
+
+    const goods = getReorderedGoods(goodsFromServer, this.state);
+
+    return (
+      <div className="section content">
+        <div className="buttons">
+          <button
+            type="button"
+            className={classNames(
+              'button',
+              'is-info',
+              {
+                'is-light': sortType !== SortType.ALPHABET,
+              },
+            )}
```

Comment:
you want to have it maximally concise and clear
```suggestion
            className={classNames('button 'is-info', {
               'is-light': sortType !== SortType.ALPHABET,
             })}
```